src/porepy/models/solution_strategy.py

```diff
@@ -520,7 +520,7 @@ def check_convergence(
         Parameters:
             nonlinear_increment: Newly obtained solution increment vector
             residual: Residual vector of non-linear system, evaluated at the newly
-            obtained solution vector.
+            obtained solution vector. Potentially None, if the residual is not needed
```

Comment:
```suggestion
            obtained solution vector. Potentially None, if the residual is not needed.
```
Isn't there also a lack of indentation for this line?

Reply:
You are right, I will fix this in the end.